content/en/docs/ops/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -72,6 +72,8 @@ $ helm upgrade istiod istio/istiod -n istio-system
 
 The ztunnel DaemonSet is the L4 node-proxy component of ambient.
 
+The ztunnel at version 1.x is compatible with control plane at version 1.x-1, 1.x, and 1.x+1, which means the ztunnel DaemonSet and the Istio control plane can be upgraded independently and in any order, as long as their version difference is within one minor version.
```

Comment:
This might be controversial, but we need to say _something_, and with inpod, ztunnel and CNI need to be (largely, for now) in lockstep with each other in terms of upgrade skew range.

Reply:
Overall PR looks fine to me.  This part may need some discussion, though. With inpod stuff, can we really say ztunnel is compatible with N-1.. IE, would 1.20's ztunnel function as expected with a 1.21 control plane or vice versa?

cc: @howardjohn what do you think here?

Reply:
I can be more explicit and say that ztunnel and CNI must be upgraded together, and should as a unit respect the version skew, rather than saying that each one individually does.

Reply:
🤔 probably (at least more) OK for alpha feat but definitely something to consider more thoroughly for Beta. Seems like a safer way forward if we can tie the components together in lock step but we'll want to consider failure modes and how they get bubbled up especially since failing to maintain this lock step might be the difference between "the ambient mesh is working" and "the ambient mesh is failing (hopefully at least very loudly)"

Reply:
I am a bit worried about making any compatibility claims. Part of being "alpha" means we can make breaking changes. In particular, this isn't even true right now, as N-1 doesn't work (N-1 had no 'inpod' at all :slightly_smiling_face: ), and I am worried about committing to N+1 support since we have no feedback on inpod yet.

Maybe we can say we strive to support this range, but given its feature status this is best effort? Later, we can make a more concrete commitment
I would say "no skew allowed" but that is hard to do

Reply:
Not sure why we need to support N-1 and add more test matrix for us vs just telling users to update Istiod (control plane) first, then data plane (ztunnel and waypoints).

Reply:
I'd in favor of dropping N-1, note, we'll likely be asked to support N, N+1 and N+2 CPs because istio supports skip version update.